plugins/basicstyles/lang/de.js

```diff
@@ -6,7 +6,7 @@ CKEDITOR.plugins.setLang( 'basicstyles', 'de', {
 	bold: 'Fett',
 	italic: 'Kursiv',
 	strike: 'Durchgestrichen',
-	subscript: 'Tiefgestellt',
+	subscript: 'Index',
```

Comment:
We shouldn't change the translation of 3rd party plugins

Reply:
done